src/main/java/org/broadinstitute/dsm/route/participant/GetParticipantRoute.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.dsm.route.participant;
+
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.model.ParticipantWrapper;
+import org.broadinstitute.dsm.security.RequestHandler;
+import org.broadinstitute.dsm.statics.RoutePath;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.broadinstitute.dsm.util.ParticipantUtil;
+import org.broadinstitute.dsm.util.UserUtil;
+import spark.QueryParamsMap;
+import spark.Request;
+import spark.Response;
+
+public class GetParticipantRoute extends RequestHandler {
+
+    @Override
+    protected Object processRequest(Request request, Response response, String userId) throws Exception {
+
+        QueryParamsMap queryParamsMap = request.queryMap();
+
+        String requestUserId = queryParamsMap.get(UserUtil.USER_ID).value();
+        if (!userId.equals(requestUserId)) throw new IllegalAccessException("User id: " + userId  + "does not match to request user id: " + requestUserId);
+
```

Comment:
I think we should also check if the user is allowed to see pt list (same roles as in FilterRoute)